packages/astro/components/ViewTransitions.astro

```diff
@@ -30,6 +30,21 @@ const { fallback = 'animate' } = Astro.props as Props;
 		!!document.querySelector('[name="astro-view-transitions-enabled"]');
 	const triggerEvent = (name: Events) => document.dispatchEvent(new Event(name));
 	const onPageLoad = () => triggerEvent('astro:page-load');
+	const announce = () => {
+		let div = document.createElement('div');
```

Comment:
Are we going to create this div every time? Is there a risk where this div is created more than once in the same page?

Reply:
Yeah, the incoming page wipes out the previous one, so it won't be retained between page loads.